src/app/components/login/verification/verification.component.html

```diff
@@ -0,0 +1,157 @@
+<div class="login-background">
+  <div class="login-container">
+    <div class="row logo-container">
+      <mat-icon class="logo col-3">outdoor_grill</mat-icon>
+      <div class="col logo-text">{{ "appName" | translate }}</div>
+    </div>
+    <div class="row login-2fa-label">
+      {{ "verificationLabel" | translate }} <b>{{ user.email }}</b>
```

Comment:
Perhaps you should remove `<b></b>` and set `font-weight` in `login-2fa-label` class.